packages/lib/server/repository/user.ts

```diff
@@ -72,6 +72,29 @@ const userSelect = Prisma.validator<Prisma.UserSelect>()({
 });
 
 export class UserRepository {
+  static async findUserByIdWithOptionalSelect({
```

Comment:
NIT: I think we can reuse `findById` here instead of creating this new function

Reply:
addressed